src/components/Proposal/ActionButton.tsx

```diff
@@ -309,20 +309,22 @@ class ActionButton extends React.Component<IProps, IState> {
     gpRewards.tokensForStaker = gpRewards.tokensForStaker || new BN(0);
     gpRewards.reputationForProposer = gpRewards.reputationForProposer || new BN(0);
 
-    Analytics.track("Redeem", {
+    Analytics.track("Redeem", Object.assign({
       "DAO Address": daoState.address,
       "DAO Name": daoState.name,
       "Proposal Hash": proposal.coreState.id,
       "Proposal Title": proposal.coreState.title,
       "Plugin Address": proposal.coreState.plugin.entity.coreState.address,
       "Plugin Name": proposal.coreState.plugin.entity.coreState.name,
+      "GEN for staking": fromWei((gpRewards.daoBountyForStaker as BN).add(gpRewards.tokensForStaker)),
```

Comment:
why this moved ?

Reply:
No special reason. I think it's just more readable when the condition is in the end of the Object.assign.